fix: salutation gender select (#3448)
plentymarkets/plugin-ceres: meta/documents/changelog_en.md

### Fixed

- On devices with iOS versions <= 12.x, certain elements were not loaded when SSR was disabled and ShopBuilder contents were used. We would like to thank @KarolKski for the contribution.

> - When entering an address, changing the salutation from, or to the "Please select" option reset all other address fields. This has been fixed.

Reviewer: ```suggestion
- When entering an address, changing the salutation from the "Please select" option reset all other address fields. This has been fixed.
```

> - While entering the address, clicking the option "please select" on the salutation field would reset all other address fields, This has been fixed.

Reviewer: ```suggestion
- When entering an address, changing the salutation from, or to the "Please select" option reset all other address fields. This has been fixed.
```
